Tasks/ServiceFabricUpdateManifests/task.json

```diff
@@ -16,14 +16,26 @@
         "Cmd"
     ],
     "version": {
-        "Major": 1,
-        "Minor": 1,
-        "Patch": 12
+        "Major": 2,
+        "Minor": 0,
+        "Patch": 0
     },
     "minimumAgentVersion": "1.95.0",
-    "instanceNameFormat": "Update Service Fabric App Versions",
+    "instanceNameFormat": "Update Service Fabric Manifests ($(updateType))",
     "inputs": [
         {
+            "name": "updateType",
+            "type": "pickList",
+            "label": "Update Type",
+            "defaultValue": "Manifest versions",
+            "required": true,
+            "helpMarkDown": "Specify the type of update that should be made to the manifest files.",
```

Comment:
can we give a hint here to users that they might have to use task multiple times for different upgrade types